Ginger/GingerCore/ALM/Qtest/ExportToQtest.cs

```diff
@@ -0,0 +1,594 @@
+
+                                            //Attaching Zip file - finish
+                                            System.IO.File.Delete(zipFileName);
+                                        }
+                                    }
+
+
+                                    //create run with activities as steps
+                                    run.CopyDesignSteps();
+                                    run.Post();
+                                    StepFactory stepF = run.StepFactory;
+                                    List stepsList = stepF.NewList("");
+
+                                    //int i = 0;
+                                    int index = 1;
+                                    foreach (Step step in stepsList)
+                                    {
+                                        //search for matching activity based on ID and not order, un matching steps need to be left as No Run
+                                        int stepDesignID = (stepsList[index]).Field("ST_DESSTEP_ID");
+                                        Activity matchingActivity = activities.Where(x => x.ExternalID == stepDesignID.ToString()).FirstOrDefault();
+                                        if (matchingActivity != null)
+                                        {
+                                            switch(matchingActivity.Status)
+                                            {
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed:
+                                                    step.Status = "Failed";
+                                                    List<IAct> failedActs= matchingActivity.Acts.Where(x => x.Status == Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed).ToList();
+                                                    string errors = string.Empty;
+                                                    foreach (Act act in failedActs) errors += act.Error + Environment.NewLine;
+                                                    step["ST_ACTUAL"] = errors;
+                                                    break;
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.NA:
+                                                    step.Status = "N/A";
+                                                    step["ST_ACTUAL"] = "NA";
+                                                    break;
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Passed:
+                                                    step.Status = "Passed";
+                                                    step["ST_ACTUAL"] = "Passed as expected";
+                                                    break;
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Skipped:
+                                                    //step.Status = "No Run";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)